src/fragments/lib/storage/flutter/sqlite3_wasm.mdx

```diff
@@ -0,0 +1,2 @@
+Amplify uses SQLite via the drift package. This requires a custom build of SQLite when targeting web. You will need to download a sqlite3.wasm file from the [releases](https://github.com/simolus3/sqlite3.dart/releases) of the sqlite3 pub package and put this file in your web/ folder.
+For more info, see [https://drift.simonbinder.eu/web/#drift-wasm](https://drift.simonbinder.eu/web/#drift-wasm).
```

Comment:
I believe this callout should also go under the `Web` section here: /lib/project-setup/platform-setup/q/platform/flutter/